specification/resources/resource-manager/Microsoft.Authorization/preview/2023-03-01-preview/privateLinks.json

```diff
@@ -0,0 +1,834 @@
+          "type": "string",
+          "description": "The scope of private link associations where network access is updated."
+        }
+      }
+    },
+    "PrivateLinkAssociationProperties": {
+      "type": "object",
+      "properties": {
+        "privateLink": {
+          "type": "string",
+          "description": "The rmpl Resource ID."
+        },
+        "publicNetworkAccess": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "publicNetworkAccessOptions",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PrivateLinkAssociationGetResult": {
+      "description": "Result of the request to get PLA for a MG scope. ",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkAssociation"
+          },
+          "description": "private link association information."
+        }
+      }
+    },
+    "PrivateLinkAssociationPropertiesExpanded": {
+      "type": "object",
+      "properties": {
+        "privateLink": {
+          "type": "string",
+          "description": "The rmpl Resource ID."
+        },
+        "publicNetworkAccess": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "publicNetworkAccessOptions",
+            "modelAsString": true
+          }
+        },
+        "tenantID": {
+          "type": "string",
+          "description": "The TenantID."
+        },
+        "scope": {
+          "type": "string",
+          "description": "The scope of the private link association."
+        }
+      },
+      "description": "Private Link Association Properties."
+    },
+    "PrivateLinkAssociation": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkAssociationPropertiesExpanded",
+          "description": "The private link association properties."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The plaResourceID."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The operation type."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The pla name."
+        }
+      }
+    },
+    "ResourceManagementPrivateLink": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
```

Comment:
> "properties": { [](http://example.com/codeflow?start=7&length=16)

Use [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-client-flatten](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-client-flatten) here, to avoid nested properties fields here.